jsapp/js/components/formLanding.es6

```diff
@@ -170,7 +169,7 @@ var FormLanding = React.createClass({
                   {/*<bem.FormView__label m='questions'></bem.FormView__label>*/}
                   <bem.FormView__label m='clone'>
                       <bem.FormView__link m='clone'
-                          data-version-id={item.version_id}
+                          data-version-id={item.uid}
```

Comment:
This is the list of form versions, so yes, `item` is a "version" object. And `data-version-id` matches what `saveCloneAs` expects in `mixins.es6`. I think it makes sense to leave it as version id.